dev/com.ibm.ws.security.spnego/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -58,3 +58,7 @@ includeClientGSSCredentialInSubject.desc=Specifies whether the client delegation
 authFilterRef=Authentication Filter Reference
 authFilterRef$Ref=Authentication filter reference
 authFilterRef.desc=Specifies the authentication filter reference. 
+
+disableLtpaCookie=Disable LTPA Cookie
+disableLtpaCookie.desc=Do not create an LTPA cookie during processing of the SPNEGO token. The LTPA Cookie does not add to the HTTP response.
```

Comment:
`The LTPA Cookie does not add to the HTTP response.`

does this mean that the LTPA is not included in the response? In the current sentence, it's not clear what is not being added. Is it correct to say

`No LTPA cookie is added to the HTTP response.`